src/test/java/io/github/jhipster/CoverageTest.java

```diff
@@ -0,0 +1,67 @@
+package io.github.jhipster;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.reflect.ClassPath;
+
+public class CoverageTest {
```

Comment:
can you add some docs here on what the class does

Reply:
I figured the name of the class would be self-explanatory but maybe not :-)

Actually maybe this test is overly pedantic, it traverses the packages and asserts that every regular class has a `...Test` companion.

I’ll await the feedback here, if this test stays I will add a comment about what it does.

Reply:
i'm not in favor of doing this ourself, why not us a gradle plugin for that?

Reply:
Well, Maven. I’m not too familiar with jacoco but if we can make it fail the build if coverage drops significantly, sure.

But don’t read too much into this particular class, it started as more of a todo list for this PR and only at the end did I add the commented assert as an afterthought

No problem to drop this commit.

Reply:
Then lets remove this. And I don't want builds to fail if coverage drops not yet atleast.